src/jose/algorithms/signing/ed25519.ts

```diff
@@ -0,0 +1,45 @@
+import * as ed25519 from '@noble/ed25519';
+import { base64url } from 'multiformats/bases/base64';
+
+import type { PrivateJwk, PublicJwk, Signer  } from '../../types';
+
+function validateKey(jwk: PrivateJwk | PublicJwk): void {
+  if (jwk.kty !== 'OKP' || jwk.crv !== 'Ed25519') {
+    throw new Error('invalid jwk. kty MUST be OKP. crv MUST be Ed25519');
+  }
+}
+
+export const Ed25519: Signer = {
```

Comment:
Can we export with lower case since it's an object? Same for `Secp256k1`:

`Ed25519` -> `ed25519`
`Secp256k1` -> `secp256k1`

Reply:
done